tsconfig.json

```diff
@@ -2,12 +2,21 @@
   "compilerOptions": {
     "target": "es2016",                                  /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
     "module": "commonjs",                                /* Specify what module code is generated. */
-    "rootDir": "./src",                                  /* Specify the root folder within your source files. */
-    "declaration": true,                              /* Generate .d.ts files from TypeScript and JavaScript files in your project. */
-    "outDir": "./dist",                                   /* Specify an output folder for all emitted files. */
+    "rootDir": "./",                                     /* Specify the root folder within your source files. */
```

Comment:
Afraid this will not work since structure of `dist/` will be changed. Currently it contains everything inside `src`, after this change it will contain `src/` and `examples/` dir. I believe you should use `include` option instead of changing `rootDir`.